parsec/backend/postgresql/user_queries/find.py

```diff
@@ -29,25 +29,29 @@ def _q_factory(query, omit_revoked):
 
 
 @lru_cache()
-def _q_human_factory(query, omit_revoked, omit_non_human):
+def _q_human_factory(query, omit_revoked, omit_non_human, offset, limit):
     conditions = []
     if query:
-        conditions.append("AND CONCAT(human.label,human.email,user_.user_id) ~* $query")
+        conditions.append("AND CONCAT(human.label,human.email) ~* $query")
     if omit_revoked:
         conditions.append("AND (user_.revoked_on IS NULL OR user_.revoked_on > $now)")
-    if omit_non_human:
+    if omit_non_human or query:
         conditions.append("AND user_.human IS NOT NULL")
     return Q(
         f"""
 SELECT
     user_.user_id,
     human.email,
     human.label,
-    user_.revoked_on IS NOT NULL AND user_.revoked_on <= $now
-FROM user_ LEFT JOIN human ON user_.human=human._id
-WHERE
+    user_.revoked_on IS NOT NULL AND user_.revoked_on <= $now,
+    count(*) OVER() AS full_count
+    FROM user_ LEFT JOIN human ON user_.human=human._id
+    WHERE
     user_.organization = { q_organization_internal_id("$organization_id") }
     { " ".join(conditions) }
+    ORDER BY user_id
```

Comment:
I'm pretty sure this is broken:

users_id | human_handle
---------|--------------
1            | bob3
2            | bob2
3            | bob1

with `query=bob`, we retrieve the 3 results, but if we ask for a 2 items in the pagination, we will end up with `1, bob3` & `2, bob2`, so we will display to the user:
First page:
- bob3
- bob2

Second page:
- bob1

Reply:
I think we shouldn't sort when asking without query parameter (no sort means we should get the results in the insertion order which should be in line with what is done in the memory implementation)

When asking with query parameter, we should only do the sort on sort on `human.label`